include/nandsettings.h

```diff
@@ -75,6 +75,7 @@ struct NandSettings
     bool hitboxNpc;
     bool hitboxNpcDisabled;
     bool lastKey;
+    bool noInventoryBounds;
```

Comment:
nandSettingsDefaults and the updating section of evt_nandsettings_handle_read_output need to be updated to handle this field
If you move noInventoryBounds to be at the end of this struct, then updating is just a case of initialising the field since all the other fields will still be in their correct place